README.md

```diff
@@ -10,3 +10,5 @@ To get started:
   - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
 
 Then follow the on-screen instructions to set up your repo.
+
+This repo contains Yenessa's portfolio and SPS projects.
```

Comment:
You don't have to keep all the stuff above this unless you want to. If it's handy to keep in your README, feel free to do so.